### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by PrantoshB**
### In the mobile version, we've implemented the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.

- When the user clicks (or taps) the close (X) button, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, that particular section of the page is displayed.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const linkHello = document.querySelector('#hellolink')
+const linkAbout = document.querySelector('#aboutlink')
+const linkPortfolio = document.querySelector('#portfoliolink')
+const linkContact = document.querySelector('#contactlink')
+/* eslint-disable no-unused-vars */
+function onClickMenu () {
```
Comment (andgarzonmal) [inline script.js:6]: - [ ] Kyndly remove the hamburguer menu button on desktop size. this is occurring when you click on the hamburger menu and then you rezize the web page.

image
 

https://user-images.githubusercontent.com/95243996/201686178-c2115a8c-e1b3-46e3-9070-978476b3712b.mp4

Comment (andgarzonmal) [inline script.js:6]: - [ ] Kyndly remove the hamburguer menu button on desktop size. this is occurring when you click on the hamburger menu and then you rezize the web page.

image
 

https://user-images.githubusercontent.com/95243996/201686178-c2115a8c-e1b3-46e3-9070-978476b3712b.mp4

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @   ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @PrantoshB 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**